test_env_vars.yaml

```diff
@@ -74,10 +74,17 @@ cached_dependencies:
     https_url: "https://github.com/release-engineering/retrodep.git"
   # Repository that will be used for the test
   test_repo:
+    # Use local bare remote repository
+    # If this is used, the git urls below should be paths and should be available for the workers
+    use_local: True
     # The tracked repository (remote)
-    ssh_url: "git@github.com:user/cachito_test_repo.git"
+    ssh_url: "./tmp/cachito-archives/integration-tests-git-repo.git"
```

Comment:
How would this change affect running tests in jenkins?

Reply:
It should not unless test_env["cached_dependencies"]["test_repo"]["use_local"] is set there.

Reply:
We are definitely not mixing both work but as I mentioned above, it is important to acknowledge the concerns. Let me know if you prefer discussing this in JIRA or someplace else.

@ejegrova Could you please help confirm this as you are familiar with how we run tests locally as well as within jenkins. (whether this change needs additional work downstream)

Reply:
I am not entirely sure if I understand the discussion. These changes are needed so the tests will run locally without having to configure github repo, etc. Jenkins configuration is something that will still need to happen because this test is causing our issues on Jenkins. But we definitely shouldn't change upstream so it will fit our needs downstream.